tests/functional/adapter/list_relations_tests/test_show_objects.py

```diff
@@ -87,3 +103,21 @@ def test_list_relations_without_caching(self, project):
             len([relation for relation in relations if relation.is_dynamic_table])
             == self.dynamic_tables
         )
+
+
+class TestShowIcebergObjects(ShowObjectsBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {"flags": {"enable_iceberg_materializations": True}}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"my_model.sql": _MODEL_ICEBERG}
+
+    def test_quoting_ignore_flag_doesnt_break_iceberg_metadata(self, project):
+        """We inject the QUOTED_IDENTIFIERS_IGNORE_CASE into the underlying query that fetches
```

Comment:
When creating a test case for a specific bug, I like linking to the originating issue in the docstring. While git blame will do some of that for you, it makes it very clear why we're testing this particular functionality.

Reply:
Can do!